packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -41,7 +42,7 @@ const Wrapper = styled.div`
  * another div inside. This implementation mirrors the current orbit banner.
  */
 const CenterWrapper = styled.div`
-  max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
+  max-width: 36.25rem;
```

Comment:
Just was trying to find where you derived this width from, looks like it come from here on the design:
<img width="650" alt="image" src="https://user-images.githubusercontent.com/9645462/104757379-e83b4580-5754-11eb-9ab2-7d995f569385.png">
582px to rem on a few website seems to come out at a slightly different value: `36.375rem`: https://www.easyunitconverter.com/px-to-rem

Don't see that as a huge issue though.

Just wondered if one of our UXers can point to where they derived the width, if it is defined somewhere in GEL it would be good to use an existing constant rather than a hardcoded width value.

Reply:
https://www.bbc.co.uk/gel

Reply:
Odd, the body text is of `width: 580px;`. The header here is positioned 2px to the left of the body text and the button.

Reply:
I spoke to Gail about this, the designs have been updated to `599px`, which is `GEL_GROUP_2_SCREEN_WIDTH_MAX` 👍